HW7/kraievskyiv/kraievskyiv_hw7.py

```diff
@@ -0,0 +1,83 @@
+#Task 1
+def largest_of_two():

```

Comment:
It would be better to create  function - largest(a, b) and after that use it in function main()